web/controllers/user_controller.ex

```diff
@@ -34,6 +35,23 @@ defmodule CodeCorps.UserController do
     end
   end
 
+  def github_connect(conn, code) do
+    current_user = Guardian.Plug.current_resource(conn)
+    with {:ok, user} <- Github.connect(current_user, code)
```

Comment:
@begedin So I was referring to the documentation [here](https://developer.github.com/v3/oauth/#2-github-redirects-back-to-your-site).

I think I may be missing something though.